soccer/src/soccer/coach/coach_node.cpp

```diff
@@ -0,0 +1,140 @@
+#include "coach_node.hpp"
+
+CoachNode::CoachNode(const rclcpp::NodeOptions& options) : Node("coach_node", options) {
+    coach_pub_ =
+        this->create_publisher<rj_msgs::msg::CoachStateInterpretation>("/strategy/coach", 10);
```

Comment:
The coach will also eventually publish position assignments, so it might be better to have this topic name be `strategy/coach_state` or something like that. Then coach will pub `strategy/role_assignments` as well and it's unambiguous which is which.

Reply:
rename the coach_pub_ var to coach_state_pub_ if this is implemented too